### PR TITLE
Fix #537 :nc focus issue

### LINKED ISSUE #537: Lost focus when using next/previous counterpart [$15]
Every time I use next counterpart (:nc) or previous counterpart shortcut (:pc), I lose focus on the current window and I have to use my mouse to get back to it (Xcode5 + Mac 10.9). It worked fine before but not anymore after I update XVim. ctrl+shift+up or ctrl+shift+down still focus on the current window.

Hope there will be a way to fix it, thank you.

<bountysource-plugin>

---
There is a **[<total-bounty>$15</total-bounty> open bounty](https://www.bountysource.com/issues/1446119-lost-focus-when-using-next-previous-counterpart?utm_campaign=plugin&utm_content=tracker%2F252770&utm_medium=issues&utm_source=github)** on this issue. Add to the bounty at [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F252770&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by pebble8888**

### CODE REVIEW

File: XVim/XVimExCommand.m [hunk 1]
```diff
@@ -1045,7 +1045,7 @@ - (void)ncounterpart:(XVimExArg*)args inWindow:(XVimWindow*)window{
     // We must make forcus back to editor first then invoke the command.
     // This is because I do not know how to move focus on newly visible text editor by invoking this command.
     // Without this focus manipulation the focus after the command does not goes to the text editor
-    //[window setForcusBackToSourceView];
+    [window setForcusBackToSourceView];
```
Comment (JugglerShu) [inline XVim/XVimExCommand.m]: Does this mean that you comment in the commented out line?
I don't remember why I commented out the line but there should be a reason...

**Comment by JugglerShu** [comment]
The log says that the reason I commented out is because when switching to the counterpart file, the cursor position always goes to 0.
However, I cannot reproduce this behaviour now. So I merge this.
Let me know if you find something wrong related to the problem.

**Comment by pebble8888** [comment]
Thanks for merge.

I'm glad to see you here again.
How are you feeling?

2015年2月11日水曜日、JugglerShu<notifications@github.com>:

**Comment by JugglerShu** [comment]
I'm OK now. I finally can work on XVim.

**Pull request merged**